rancid-tomatillos/src/App/App.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from "react";
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    Redirect,
+} from "react-router-dom";
+import Dashboard from "../Dashboard/Dashboard";
+import Header from "../Header/Header";
+import Login from "../Login/Login";
+import ShowPage from "../MoviePage/MoviePage";
+
+import "./App.css";
+
+class App extends Component {
+    constructor() {
+        super();
+        this.state = {
+            loggedIn: false,
+            movies: null,
+            error: "",
+        };
+    }
+    async componentDidMount() {
+        let promise = await fetch(
+            "https://rancid-tomatillos.herokuapp.com/api/v2/movies/"
+        );
+        if (promise.ok) {
+            let result = await promise.json();
+            this.setState({movies: result.movies});
+        } else {
+            this.setState({error: promise.status});
```

Comment:
added a low priority issue card to bulk up the error a bit so it reads something like (404 status text) instead of just the error #